src/commands/build.js

```diff
@@ -17,6 +18,11 @@ module.exports = function() {
     const statsJson = stats.toJson();
     fs.writeFileSync("webpack-bundle-stats.json", JSON.stringify(statsJson));
     fs.writeFileSync("build/asset-path.json", JSON.stringify({assetPath: getAssetPath()}));
+
+    if (staticBuild) {
+      generateStaticHTMLFile();
+    }
```

Comment:
nothing too fancy going on here… during the build process, if `--static` was passed then we are going to have it build out the static html file.